docs/1.0-Installation.md

```diff
@@ -1,20 +1,37 @@
 # 1.0 Installation
 
+Quick Access:
+- [1.0 Installation](#10-installation)
+    - [1.0.1 - Glossary](#101---glossary)
+  - [1.1 - Prerequisites](#11---prerequisites)
+    - [1.1.1 - IAM User and Secret](#111---iam-user-and-secret)
+    - [1.1.2 AWS Accounts](#112-aws-accounts)
+    - [1.1.3 STS Roles](#113-sts-roles)
+      - [1.1.3.1 - Jump Role](#1131---jump-role)
+      - [1.1.3.2 - Access Role](#1132---access-role)
+    - [1.1.4 Config Map](#114-config-map)
+  - [1.2 Workflow](#12-workflow)
+  - [1.3 Testing your AWS account credentials with the CLI](#13-testing-your-aws-account-credentials-with-the-cli)
+  - [1.4 Development](#14-development)
+  - [1.5 Operator Install](#15-operator-install)
+    - [1.5.1 Local Mode](#151-local-mode)
+    - [1.5.2 Cluster Mode](#152-cluster-mode)
+
 ### 1.0.1 - Glossary
 
 Terms used in this guide and what they mean for this guide:
 
 * Payer Account - AWS Organizations give you a "unified bill" so to speak under a single account. Sub accounts are created under this "Payer Account", which is the highest-level account in this organization. SREs all should also have an IAM user with credentials on each payer account, which has AssumeRole Permissions as administrator into each of these sub-accounts.
-* Jump Role - Using AWS Assume-Role Chaining, a Jump Role is an intermediary role that sits between a given IAM user and a customer's access role, providing a single point of entry.
+* Jump Role - Using AWS Assume-Role Chaining, a `Jump Role` is an intermediary role that sits between a given IAM user and a customer's `Access Role`, providing a single point of entry.
 * Access Role - A role inside the cluster's account that we assume to perform tasks.
```

Comment:
Can we move this section to the glossary page?

Reply:
Done